org.o20.word.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "org.o20.word",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "word64.run",
+    "tags": ["beta"],
+    "desktop-file-name-suffix": " Beta",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--filesystem=host" ],
```

Comment:
What does it need host access for? Does it not use portals for opening/saving files?